modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -99,123 +95,340 @@ Status description:: error: connection exception - alias chains are not permitte
+=== 25N11
+
+Status description:: error: invalid transaction state - conflicting transaction state. There was a conflict detected between the transaction state and applied updates. Please retry the transaction.
+
+=== 25N12
+
+Status description:: error: invalid transaction state - index was dropped. Index { $idx } was dropped in this transaction and cannot be used.
+
+=== 25N13
+
+Status description:: error: invalid transaction state - cannot access entity after removal. A { $entityType } was accessed after being deleted in this transaction. Verify the transaction statements.
+
+== Invalid transaction termination
+
+Invalid transaction termination errors occur when the transaction termination fails, such as when the transaction or constituent transaction fails to commit, or when the transaction termination fails to apply or append the transaction.
+
+=== 2DN01
+
+Status description:: error: invalid transaction termination - commit failed. Failed to commit transaction. See debug log for details.
+
+=== 2DN02
+
+Status description:: error: invalid transaction termination - constituent commit failed. Failed to commit constituent transaction. See debug log for details.
+
+=== 2DN03
+
+Status description:: error: invalid transaction termination - transaction termination failed. Failed to terminate transaction. See debug log for details.
+
+=== 2DN04
+
+Status description:: error: invalid transaction termination - constituent transaction termination failed. Failed to terminate constituent transaction. See debug log for details.
+
+=== 2DN05
+
+Status description:: error: invalid transaction termination - failed to apply transaction. There was an error on applying the transaction. See logs for more information.
+
+=== 2DN06
+
+Status description:: error: invalid transaction termination - failed to append transaction. There was an error on appending the transaction. See logs for more information.
+
+=== 2DN07
+
+Status description:: error: invalid transaction termination - inner transactions still open. Unable to commit transaction because it still have non-closed inner transactions.
+
+[[transaction-rollback]]
+== Transaction rollback
+
+Transaction rollback errors occur when the transaction is rolled back due to an error, such as when the transaction rollback fails, or when the constituent rollback fails.
+
+=== 40N01
+
+Status description:: error: transaction rollback - rollback failed. Failed to rollback transaction. See debug log for details.
+
+=== 40N02
+
+Status description:: error: transaction rollback - constituent rollback failed. Failed to rollback constituent transaction. See debug log for details.
+
+[[syntax-error-access-rule-violation]]
+== Syntax error or access rule violation
+
+Syntax error or access rule violation errors occur when there is a syntax error in the query, or when the query violates the access rules, such as when the query contains an invalid syntax, or when the query tries to access a database that does not exist, etc.
```

Comment:
```suggestion
Syntax error or access rule violation errors occur when a Cypher query contains invalid syntax or when a client request violates the access rules, such as when a query tries to access a database without enough privileges, etc.
```

I found your sentence here a bit repetitive, and also I am unsure about the database not exist example as that also occurs under another class. So I tried to reformulate it. Feel free to improve my English or to go with something in between our descriptions if you like.